independent-publisher-connectors/OpenTriviaDB Proposal- OTDB (Independent Publisher)/apiDefinition.swagger.json

```diff
@@ -0,0 +1,201 @@
+        "operationId": "GetQuestion",
+        "parameters": [
+          {
+            "name": "amount",
+            "in": "query",
+            "required": true,
+            "type": "integer",
+            "description": "Amount",
+            "x-ms-summary": "Amount of questions"
+          },
+          {
+            "name": "category",
+            "in": "query",
+            "required": false,
+            "type": "integer",
+            "description": "Category_ID",
+            "x-ms-summary": "Category_ID",
+            "enum": [
+              9,
+              10,
+              11,
+              12,
+              13,
+              14,
+              15,
+              16,
+              17,
+              18,
+              19,
+              20,
+              21,
+              22,
+              23,
+              24,
+              25,
+              26,
+              27,
+              28,
+              29,
+              30,
+              31,
+              32
+            ]
+          },
+          {
+            "name": "difficulty",
+            "in": "query",
+            "required": false,
+            "type": "string",
+            "description": "Difficulty",
+            "x-ms-summary": "Difficulty",
+            "enum": [
+              "easy",
+              "medium",
+              "hard"
+            ]
+          },
+          {
+            "name": "type",
+            "in": "query",
+            "required": false,
+            "type": "string",
+            "enum": [
+              "boolean",
+              "multiple"
+            ],
+            "description": "Type",
+            "x-ms-summary": "Type of question"
+          }
+        ]
+      }
+    },
+    "/api_count.php": {
+      "get": {
+        "responses": {
+          "default": {
+            "description": "default",
+            "schema": {}
+          }
+        },
+        "summary": "Question Count Lookup",
+        "description": "Returns the number of questions in the database, in a specific category.",
+        "operationId": "QuestionCountLookup",
+        "parameters": [
+          {
+            "name": "category",
+            "in": "query",
+            "required": false,
+            "type": "integer",
+            "enum": [
+              9,
+              10,
+              11,
+              12,
+              13,
+              14,
+              15,
+              16,
+              17,
+              18,
+              19,
+              20,
+              21,
+              22,
+              23,
+              24,
+              25,
+              26,
+              27,
+              28,
+              29,
+              30,
+              31,
+              32
+            ],
+            "description": "Category",
+            "x-ms-summary": "Category"
+          }
+        ]
+      }
+    },
+    "/api_count_global.php": {
+      "get": {
+        "responses": {
+          "default": {

```

Comment:
@Kiveshan, default responses are not allowed. Please define a proper response. please see our docs [here](https://learn.microsoft.com/en-us/connectors/custom-connectors/coding-standards#responses) and [here](https://learn.microsoft.com/en-us/connectors/custom-connectors/connector-string-guidance#operations)
Please fix all responses in the swagger file